### PR TITLE
Inline docs

### PR DESCRIPTION
**Pull request by 0aveRyan**
* Inline docblocks
* CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* If possible, and if applicable, please also add/update unit tests for your changes
+* Push the changes to your fork and submit a pull request to the 'develop' branch of this repository
+
+## Code Documentation
+
+* We strive for full doc coverage and follow the standards set by phpDoc
+* An example function can be found [here](https://gist.github.com/sunnyratilal/5308969)
+* Please make sure that every function is documented so that when we update our API Documentation things don't go awry!
+	* If you're adding/editing a function in a class, make sure to add `@access {private|public|protected}`
+* Finally, please use tabs and not spaces. The tab indent size should be 4 for all EDD code.
```
Comment (CodeProKid) [inline CONTRIBUTING.md]: Remove the reference to "EDD" here.

File: src/Router.php [hunk 2]
```diff
@@ -384,6 +384,9 @@ public static function process_http_request() {
 		 * to hook in to track metrics, such as how long the process took from `graphql_process_http_request`
 		 * to here, etc.
 		 *
+		 * @param $response array
+		 * @param $graphql_results array
```
Comment (CodeProKid) [inline src/Router.php]: The type should actually come before the name of the variable for our inline doc parsing to pick up the type.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* Make the changes to your forked repository
+  * Ensure you stick to the [WordPress Coding Standards](https://codex.wordpress.org/WordPress_Coding_Standards)
+* When committing, reference your issue (if present) and include a note about the fix
+* If possible, and if applicable, please also add/update unit tests for your changes
+* Push the changes to your fork and submit a pull request to the 'develop' branch of this repository
+
+## Code Documentation
+
+* We strive for full doc coverage and follow the standards set by phpDoc
+* An example function can be found [here](https://gist.github.com/sunnyratilal/5308969)
```
Comment (jasonbahl) [inline CONTRIBUTING.md]: also this reference to Sunny

**Comment by 0aveRyan** [comment]
@CodeProKid @jasonbahl rtg

**Pull request merged**